requirements.txt

```diff
@@ -5,7 +5,7 @@ django-cors-headers==3.13.0
 flojoy
 matplotlib==3.5.2
 networkx==2.8.4
-numpy==1.23.0
+numpy==1.24.3
```

Comment:
should this be updated for everyone? if so, please make a note on why?

Reply:
Not sure